// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`produces very verbose output by default 1`] = `
"
    successful test
    failing test Error thrown in test

  1 test failed

  failing test

  <FILEPATH HIDDEN>:12

   11:   when: x = 3;          
   12:   then: x === double(2);
   13: });                     

  Error thrown in test:

  AssertionError [ERR_ASSERTION] (AssertionError) {
    actual: false,
    code: 'ERR_ASSERTION',
    expected: true,
    generatedMessage: false,
    operator: '==',
    powerAssertContext: {
      args: [
        Object { … },
      ],
      source: {
        ast: Object { … },
        content: '_assert(x === double(2), \\"(x === double(2)) is not truthy\\")',
        filepath: 'ava.js',
        line: 12,
        tokens: Array [ … ],
        visitorKeys: Object { … },
      },
    },
    message: \`(x === double(2)) is not truthy   # ava.js:12␊
      ␊
      _assert(x === double(2), \\"(x === double(2)) is not truthy\\")␊
              | |   |                                            ␊
              | |   4                                            ␊
              3 false                                            ␊
      ␊
      [number] double(2)␊
      => 4␊
      [number] x␊
      => 3␊
      \`,
  }

"
`;
